src/Single/LargeSinglePlainTextFormatter.php

```diff
@@ -86,7 +86,7 @@ private function formatMoreDays(DateTimeInterface $startDate, DateTimeInterface
         return PlainTextSummaryBuilder::start($this->translator)
             ->from($formattedStartDayOfWeek, $formattedStartDate)
             ->at($formattedStartTime)
-            ->till($formattedEndDayOfWeek, $formattedEndDate)
+            ->tillLarge($formattedEndDayOfWeek, $formattedEndDate)
```

Comment:
This is already added in a previous PR, but when I look into more detail at `PlainTextSummaryBuilder` it makes no references to sizes. So this should have a name without referring to the size. The same applies to the translation key `till_large`. That is the only key that refers to the size.

Reply:
`till_included` may be cleaner as the key in this case. But to be fair the translator is very badly designed by the original authors of this library. Any modern translator will work with translations for complete sentences, not individual words. And we are running into the limitations of a translator that works with individual words now, as we need to translate the same word `till` into two different translations in 2 separate contexts.

Reply:
The `PlainTextSummaryBuilder` that I added a while back is also just a workaround to make it easier to construct complete sentences using the existing translator 🤷‍♂️

Reply:
I think replacing this file with a modern translation library would be a good idea. But I think it would be better to plan that after the changes are finalised. These changes were asked by marketing after seeing previous changes in production. Once they're happy I would plan to replace the current Translation Library. As for now, I've changed the keyname.